Automattic/woocommerce-payments
# feat: add WCPay sections in settings pages

Part of https://github.com/Automattic/woocommerce-payments/issues/1576

#### Changes proposed in this Pull Request









Adds the "WooCommerce Payments" and the "All payment methods" sections to the "Payments" tab in WC settings
https://d.pr/i/MrWNFZ

Also ensures that when loading the "Payments" tab, the WCPay settings are displayed by default: https://d.pr/i/bxO9gL
(the page is currently blank because of https://github.com/Automattic/woocommerce-payments/pull/1551 )


#### Testing instructions





- Enable WCPay grouped settings in the dev tools
- Enable Sepa by adding a `_wcpay_feature_sepa` in `wp_options` with value `'1'` (or don't, it doesn't really matter)
- Accessing http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout loads the WCPay settings (the page is currently blank because of https://github.com/Automattic/woocommerce-payments/pull/1551 )
- Accessing http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=all-gateways loads the WC Core Payment settings page
- Saving the settings in http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=all-gateways works and shows the correct page
- By disabling the WCPay grouped settings feature in dev tools, everything goes back to normal

-------------------

Does not apply - should be behind the scenes
